tests/unit-tests/python/foglamp_test/services/core/api/test_backup_restore.py

```diff
@@ -100,29 +96,86 @@ class TestBackup:
         assert response_code == r.status
         r = r.read().decode()
         retval = json.loads(r)
-        assert output == retval
+        response_length = len(retval['backups'])
+        assert exp_length == response_length
+        count = 0
+        for i in range(response_length):
+            count += 1
+            assert retval['backups'][i]['id'] == exp_output[exp_length - count]['id']
+            assert retval['backups'][i]['status'] == Status(exp_output[exp_length - count]["status"]).name
+            assert retval['backups'][i]['date'] is not None
+        conn.close()
+
+    @pytest.mark.parametrize("request_params, response_code, output_code, output_message", [
+        ('?limit=invalid', 200, 400, "limit must be an integer"),
+        ('?skip=invalid', 200, 400, "skip must be an integer"),
+        ('?status=invalid', 200, 400, "'INVALID' not a valid status"),
+    ])
+    async def test_get_backups_negative(self, request_params, response_code, output_code, output_message):
```

Comment:
Better name could be test_get_backups_invalid_request or test_get_backups_bad_data